static/app/views/replays/details.tsx

```diff
@@ -99,6 +104,26 @@ function ReplayDetails({
     );
   }
 
+  // If the user has navigated to the replay from an event, then we want to
+  // start the video at the time of the event.
+  if (!defined(initialTimeOffset) && defined(eventTimestamp) && startTimestampMs) {
+    // check if our event timestamp is within the range of the replay
+    const eventTimestampMs = new Date(eventTimestamp).getTime();
```

Comment:
`eventTimestamp` could be a negative number here, it could be anything really not always a number. Would that cause problems?

Reply:
Added a new condition to the function in case this return a `NaN`!